fenix/app/src/test/java/org/mozilla/fenix/home/HomeMenuViewTest.kt

```diff
@@ -248,4 +248,24 @@ class HomeMenuViewTest {
 
         assertTrue(testContext.settings().openNextTabInDesktopMode)
     }
+
+    @Test
+    fun `WHEN Help menu item is tapped THEN open the browser to SUMO Help page and record metrics`() {
```

Comment:
@HarrisonOg Instead of adding a new test here, could we add the extra checks in the existing test on [Line 183](https://github.com/mozilla-mobile/firefox-android/pull/4066/files#diff-2d8312648289a4580ff8c9c6870cc7fa8e3df0e996f6e4e455321f84a56ca645R183)

Reply:
moved into the existing test.